test/unittest/xmlparser/regressions/13418.xml

```diff
@@ -0,0 +1 @@
+<types><type><typedef type=""arrayDimensions=""/></type></types>
```

Comment:
```suggestion
<types><type><typedef type=""arrayDimensions=""/></type></types>

```

Reply:
I'm not going to take this, as the regression file is the one downloaded from the oss-fuzz report.